project/chapters/models.py

```diff
@@ -23,22 +24,36 @@ class ChaptersIndexPage(Page):
     parent_page_types = [
         "home.HomePage",
     ]
-    subpage_types = ["chapters.Chapter"]
+
+    subpage_types = []
 
     max_count = 1
 
+    additional_context = "groups"
+    context_order = ("title",)
+
+    class Meta:
+        abstract = True
+
     def get_context(self, request):
         """
-        return a Queryset of Chapters
+        return a Queryset of Groups
         """
         context = super().get_context(request)
-        context["chapters"] = (
-            Chapter.objects.child_of(self).live().order_by("region", "title")
+        context[self.additional_context] = (
+            Chapter.objects.child_of(self).live().order_by(*self.context_order)
```

Comment:
This may be a typo since we are in the `GroupsIndexPage` class. It might be possible to make it a bit more generic since we have restricted the `<Group|Chapter>IndexPage` model `subpage_types` to allow only `Group` or `Chapter` both of which inherit from `Page`:

```suggestion
            Page.objects.child_of(self).live().order_by(*self.context_order)
```